Omnibus.md

```diff
@@ -3446,16 +3471,18 @@ Restore Enchantment
   spell is wasted if the quest fails or if permission for a repair is outright
   denied.
 
-### Regenerate the Scalp (5th Circle) ### {#regenerate-the-scalp}
+### Regenerate the [Soul]{.new} (5th Circle) ### {#regenerate-the-soul}
 
 __Uses:__ 1 - __Verbal:__ 30 words - __Material:__ An item similar to the
-spellcaster's scalp, with some sort of indication that it is their second scalp
-- __Caveats:__ [Regeneration](#regeneration-caveat), [Advanced
-Regeneration](#advanced-regeneration)
+spellcaster's [soul token]{.new}, with some sort of indication that it is
+their second [soul token]{.new} -
+__Caveats:__ [Regeneration](#regeneration-caveat),
+[Advanced Regeneration](#advanced-regeneration)
 
 This spell grants an advanced regeneration which [returns]{.new} a character to
-life from scalping [only]{.new}. After regenerating in this fashion, the
-material component becomes the spellcaster's new scalp token.
+life from [soullessness only]{.new}. After regenerating in this fashion, the
+material component becomes the spellcaster's new [soul]{.new} token.
```

Comment:
This is the one instance of "scalp token" in the old text that isn't in Embrace Death.

Reply:
Same as the other thread, fine either way.

Reply:
See previous thread.